packages/replay/src/coreHandlers/handleGlobalEvent.ts

```diff
@@ -22,7 +19,7 @@ export function handleGlobalEventListener(replay: ReplayContainer): (event: Even
 
     // Only tag transactions with replayId if not waiting for an error
     // @ts-ignore private
-    if (event.type !== 'transaction' || !replay._waitForError) {
```

Comment:
What's the reason for this change?

Reply:
To be explicit that we only want this when the event is an error. This is IMHO more future proof, as when/if we add further event types, we'll need to explicitly handle them.